# OpenWings OpenData on Amazon Web Services

![OpenWings Logo](img/OpenWingsProject-logo-small.png "OpenWings Logo")

The OpenWings Project is a scientific research project having the goal of inferring a time-calibrated [phylogeny]() for all of the world's ~10,000 bird species (class [Aves](https://en.wikipedia.org/wiki/Bird)).  To generate this phylogeny, we have collected DNA sequence data from genomic libraries that have been enriched for a class of genomic markers known as [ultra-conserved elements](https://en.wikipedia.org/wiki/Ultraconserved_element) or UCEs. The enrichment process effectively increases the concentration of UCE loci relative to the rest of the genome, and we sequence the enriched product.

The set of 4,434 loci targeted for this project (5Kv2A and 5Kv2B) are a slightly modified subset of UCE loci commonly used in other studies of birds.  More information on the target enrichment baits used for this study can be found in the [OpenWings Bait repository](https://github.com/openwings-project/baits).

## About the data

We are providing the "raw" reads that we have generated from genomic libraries enriched for UCE loci. It is important to note that these data **HAVE NOT** been rigorously quality controlled.  The taxonomy of the individuals sequence **HAVE NOT** been standardized.

What this means is that we think these data represent the species indicated and the loci targeted, but we have not performed significant analyses of these data to verify that this is the case. Verification is an ongoing process. This also means some species labels could be incorrect.

Taxonomies used for species names have also not been standardized. This also means some species labels could be incorrect.

## Data format

The data for each species are 150 base pair (bp), paired-end (PE), Illumina sequencing reads (i.e., PE150). Data are formatted as `fastq.gz` files, and each species in the dataset has both a R1 and a R2 file (representing each of the paired reads).  File names look like the following - e.g., the two read files for the species `Rhea pennata`:
```
B-10338.LSUMZ.TISS-3:D4.Rhea-pennata.R1.fq.gz                   
B-10338.LSUMZ.TISS-3:D4.Rhea-pennata.R2.fq.gz
```
The file naming convention is:
```
[Museum_Accession].[Museum_Code].[DNA_Plate]:[DNA_Row][DNA_Column].[Genus]-[species].[Read]
```
|                    |      |
| ------------------ | ---- |
| [Museum_Accession] | This is the accession number of the particular tissue used for DNA extraction, target enrichment, and DNA sequencing. This may correspond to a tissue sample in a collection *OR* to a specimen/skin within a given collection.|
| [Museum_Code]      | This is the INSDC code for the institution providing the tissue/toepad used for DNA extraction.  Generally speaking, the combination of the [Museum_Code] and [Museum_Accession] should be referenced every time that data from this specimen are used.|
| [DNA_Plate]        | This is the DNA extraction plate in which a particular specimen was placed.  Values of `TISS-` and `LOAN-` correspond to tissues, and values of `TP-` correspond to toepads.|
| [DNA_Row]          | This is the row location of the sample in the DNA extraction plate.|
| [DNA_Column]       | This is the column location of the sample in the DNA extraction plate.|
| [Genus]-[species]  | This is the likely species of the individual from which a tissue/toepad was collected.  The taxonomy of these individuals has not necessarily been standardized, because these values were given by the institution providing the tissue.|
| [Read]             | This is a value of `R1` for read 1 and `R2` for read 2. |

## Downloading/mirroring the data

## Data license

Data available from the OpenWings project are released under a [Creative Commons](https://creativecommons.org) [CC-0]() license.

## Data restrictions

Although the data are released under CC-0 license, the primary investigators of OpenWings request the right to publish studies using the data collected at taxonomic levels above order (e.g. interordinal and higher levels), and to infer and time-calibrate phylogenies using the entire complement of data collected from all bird species **before these data are used by other investigators for similar purposes**.  This is in keeping with the so-called [Ft. Lauderdale Agreement](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf).

## If you use these data...

Studies that use data, libraries, or other resources generated by OpenWings should acknowledge support from the following National Science Foundation grants:
|                    |
| ------------------ |
|DEB-1655559|
|DEB-1655624|
|DEB-1655683|
|DEB-1655736|


We also request that authors of research products using these resources return information to OpenWings regarding publication/presentation of their work so that we can track the impact of this research project.  We will provide easy-to-use forms for this purpose.

## Major Participating Institutions

* American Museum of Natural History (Brian Smith)
* Australian National Wildlife Collection (Leo Joseph)
* University of Minnesota / Bell Museum of Natural History (Keith Barker, Sushma Reddy)
* UW / Burke Museum of Natural History and Culture (Sharon Birks, Adam Leache, Alejandro Rico-Guevara)
* Bruce Museum (Dan Ksepka)
* Field Museum of Natural History (Ben Marks, Shannon Hackett, John Bates)
* LSU / LSU Museum of Natural Science (Brant Faircloth, Robb Brumfield)
* Moore Laboratory of Zoology (John McCormack)
* Museu de Zoologia da USP (Luís Fábio Silveira)
* National History Museum of Denmark (Peter Hosner)
* University of Florida (Rebecca Kimball, Ed Braun)
* University of Kansas Biodiversity Institute (Rob Moyle)
* NMNH, Smithsonian Institution, USGS Biological Survey Unit (Terry Chesser)
* University of Alaska Museum of the North (Kevin Winker)


